Add frequency plans to docs (#2562)
TheThingsNetwork/lorawan-stack: doc/content/reference/frequency-plans/adding-frequency-plans.md

---
title: "Adding Frequency Plans"
description: ""
---

> {{% tts %}} uses frequency plans from the [lorawan-frequency-plans Github repository.](https://github.com/TheThingsNetwork/lorawan-frequency-plans/)

Reviewer: ```suggestion
{{% tts %}} uses frequency plans from the [`lorawan-frequency-plans` GitHub repository](https://github.com/TheThingsNetwork/lorawan-frequency-plans/).
```

> To add frequency plans, you may open a pull request on the [lorawan-frequency-plans Github repository](https://github.com/TheThingsNetwork/lorawan-frequency-plans/), which makes the plan available in most deployments.

Reviewer 2: > which makes the plan available in most deployments

This is not completely accurate. Most deployments (including TTI Cloud Hosted) only pull the frequency plans when a new version is deployed. Other deployments use a fork of that repository, which will not update automatically.

> To add frequency plans, you may open a pull request on the [lorawan-frequency-plans Github repository](https://github.com/TheThingsNetwork/lorawan-frequency-plans/), which makes the plan available in most deployments when the next version is deployed.

Reviewer: ```suggestion
To add frequency plans, you may open a pull request for [`lorawan-frequency-plans`](https://github.com/TheThingsNetwork/lorawan-frequency-plans/), which makes the plan available in most deployments when the next version is deployed.
```

You may also fork the repository and reference your fork, with potentially custom plans, or with standard plans left out.

> You may also configure a local directory (or cloud storage bucket) where the frequency plans are stored, in case the private network cannot access GitHub or does not want to rely on GitHub.

Reviewer 2: This has nothing to do with "Adding Frequency Plans".

Author: These are @johanstokking 's instructions for adding frequency plans. You can fork the repo, add custom plans, or store a file with custom plans locally. How does this have nothing to do with adding frequency plans?

Reviewer: I agree this is useful
